FortnoxAPILibrary/Connectors/ArchiveConnector.cs

```diff
@@ -334,7 +283,52 @@ public void DeleteFile(string fileId)
 		/// <param name="folderId">The id of the folder to be deleted.</param>
 		public void DeleteFolder(string folderId)
 		{
-			base.BaseDelete(folderId);
+            Resource = "archive";
+
+			BaseDelete(folderId);
 		}
-	}
+
+        private static string GetMimeType(string name)
+        {
+            var extension = System.IO.Path.GetExtension(name);
+            return MimeTypeMap.GetMimeType(extension);
+        }
+
+        private static bool IsFolderId(string destination)
+        {
+            return Guid.TryParse(destination, out var unused);
```

Comment:
if variable is unused, expression can be simplified to:

`return Guid.TryParse(destination, out _);`